daffodil-cli/src/it/scala/org/apache/daffodil/cliTest/TestCLIUdfs.scala

```diff
@@ -179,105 +225,159 @@ class TestCLIUdfs {
-    } (ExitCode.UserDefinedFunctionError)
+      cli.expectErr(
+        "[error] User Defined Function 'ssudf:reverse' Error. Cause: org.sbadudfs.udfexceptions.evaluating.StringFunctions.Reverse$CustomException: UDF Error!",
+      )
+      cli.expectErr(
+        "at org.sbadudfs.udfexceptions.evaluating.StringFunctions.Reverse.evaluate(StringFunctionsProvider.scala:56)",
+      )
+    }(ExitCode.UserDefinedFunctionError)
   }
 
   /**
    * Tests the case when a function class:
    *   throws processing error on evaluate
    */
   @Test def test_UDFClass_ProcessingErrorOnEvaluate(): Unit = {
-    val schema = path("daffodil-udf/src/test/resources/org/apache/daffodil/udf/genericUdfSchema.xsd")
-    val classpath = udfClasspath("daffodil-udf/src/test/scala/org/sbadudfs/udfexceptions/evaluating/StringFunctions/")
+    val schema = path(
+      "daffodil-udf/src/test/resources/org/apache/daffodil/udf/genericUdfSchema.xsd",
+    )
+    val classpath = udfClasspath(
+      "daffodil-udf/src/test/scala/org/sbadudfs/udfexceptions/evaluating/StringFunctions/",
+    )
 
     runCLI(args"parse -s $schema -r user_func3", classpath) { cli =>
-      cli.expectErr("[error] Schema Definition Error: User Defined Function 'ssudf:rev-words' Error: UDF PE!")
-    } (ExitCode.UnableToCreateProcessor)
+      cli.expectErr(
+        "[error] Schema Definition Error: User Defined Function 'ssudf:rev-words' Error: UDF PE!",
+      )
+    }(ExitCode.UnableToCreateProcessor)
   }
 
   /**
    * Tests the case when a function class:
    *   throws an error while being loaded
    */
   @Test def test_UDFClass_exceptionOnLoad(): Unit = {
-    val schema = path("daffodil-udf/src/test/resources/org/apache/daffodil/udf/genericUdfSchema.xsd")
-    val classpath = udfClasspath("daffodil-udf/src/test/scala/org/sbadudfs/udfexceptions2/StringFunctions/")
+    val schema = path(
+      "daffodil-udf/src/test/resources/org/apache/daffodil/udf/genericUdfSchema.xsd",
+    )
+    val classpath = udfClasspath(
+      "daffodil-udf/src/test/scala/org/sbadudfs/udfexceptions2/StringFunctions/",
+    )
 
     runCLI(args"-v parse -s $schema -r user_func3", classpath) { cli =>
-      cli.expectErr("[error] User Defined Function could not be initialized: {http://example.com/scala/udf}rev-words.")
-      cli.expectErr("Cause: org.sbadudfs.udfexceptions2.StringFunctions.ReverseWords$CustomException: UDF Error!")
-      cli.expectErr("at org.sbadudfs.udfexceptions2.StringFunctions.ReverseWords.<init>(StringFunctionsProvider.scala:65)")
-    } (ExitCode.UserDefinedFunctionError)
+      cli.expectErr(
+        "[error] User Defined Function could not be initialized: {http://example.com/scala/udf}rev-words.",
+      )
+      cli.expectErr(
+        "Cause: org.sbadudfs.udfexceptions2.StringFunctions.ReverseWords$CustomException: UDF Error!",
+      )
+      cli.expectErr(
+        "at org.sbadudfs.udfexceptions2.StringFunctions.ReverseWords.<init>(StringFunctionsProvider.scala:65)",
```

Comment:
Likewise, remove the line number (or more) from this string argument to avoid `Expect operation fails (timeout: -1 ms) for matcher: contains('at org.sbadudfs.udfexceptions2.StringFunctions.ReverseWords.<init>(StringFunctionsProvider.scala:65)'`.  Being too verbose makes the test more brittle (what if the path changes due to renaming or shortening changes as well?).

Reply:
Line numbers have been removed from the tests